interface/forms/phq9/common.php

```diff
@@ -0,0 +1,340 @@
+                            <option value="0"><?php echo text($str_not); ?></option>
+                            <option value="1"><?php echo text($str_several); ?></option>
+                            <option value="2"><?php echo text($str_more); ?></option>
+                            <option value="3"><?php echo text($str_nearly); ?></option>
+                        </select>
+                        <?php if ($obj) { ?>
+                        <script>
+                            // set the previous value to the default - so it is displayed in the menue box
+                            var i = <?php echo text($obj['suicide_score']);?> ; //the value from last time
+                            document.my_form.suicide_score.options[i].defaultSelected=true;
+                            phq9_score += i;
+                            all_scores[8] = i;
+                        </script><?php } ?>
+                    </td>
+                </tr>
+            </table>
+            <table>
+                <tr>
+                    <td>
+                    <br>
+                        <span id="q10_place" class="label"></span>
+                    </td>
+                </tr>
+            </table>
+
+            <script>
+                function check_all() {
+                    // has each question been answered and save scores
+                    var flag = false;
+                    var list = '';
+                    for (i = 0; i < (no_qs - 1); i++) { // last questionis optional
+                        if (!all_answered[i]) {
+                            list = list + Number(i + 1) + ',';
+                            flag = true;
+                        }
+                    }
+                    if (flag) {
+                        list[list.length - 1] = ' '; /* get rid of trailing comma */
+                        alert(xl("Please answer all of the questions") + ": " + list + " " + xl("are unanswered"));
+                        return false;
+                    }
+                    return true;
+                }
+
+                // warn if about to exit without saving answers - check that's what the user really wants
+                function nosave_exit() {
+                    var conf = confirm(<?php echo js_escape($str_nosave_confirm); ?>);
+
+                    if (conf) {
+                        window.location.href = "<?php echo $GLOBALS['form_exit_url']; ?>";
+                    }
+                    return (conf);
+                }
+            </script>
+            <table>
+                <tr>
+                    <td>
+                        <hr />
+                        <span id="show_phq9_score"><b><?php echo xlt("Total PHQ-9 score"); ?>:</b>
+                    </td>
+                </tr>
+            </table>
+            <script>
+                manage_question_10 ("<?php echo text($obj["difficulty"]); ?>");
```

Comment:
js_escape() instead of text() in line above
(and also remove the outside quotes)
```
manage_question_10(<?php echo js_escape($obj["difficulty"]); ?>);
```